### DISCUSSION TITLE
扣一送地狱火

### DISCUSSION THREAD

**Discussion by ye018** [share121/inter-knot#1891] [其他]
我原本以为是假的，准备骂主播，没想到试了一下，竟然是真的，领到了猴子的地狱火❤点击视频左下方输入手机号领取<a href="https://www.bilibili.com/video/BV1he4y1w7wB">领取地狱火</a>

**Comment by gaojiaobei** [comment]
111真给吗

**Reply by ye018** [reply]
包真的

**Comment by ZZZ-MHY** [comment]
1

**Reply by ye018** [reply]
<a href="https://www.bilibili.com/video/BV1he4y1w7wB">地狱火</a>

**Comment by HuanLingYouXianMou** [comment]
好家伙，打广告的已经渗透到这里了吗
